src/Potato/Data/Text/Zipper2.hs

```diff
@@ -154,7 +156,11 @@ pageDown pageSize z = undefined
 
 -- | Move the cursor to the beginning of the current logical line (clearing the selection)
 home :: TextZipper -> TextZipper
-home (TextZipper lb b s a la) = undefined
+home (TextZipper lb b [] a la) = TextZipper lb "" [] (b <> a) la
+home (TextZipper lb b (x:[]) a la) = TextZipper lb "" [] (b <> x) (a:la)
```

Comment:
I don't think this case is correct. What if your zipper looks like this ([] is where the cursor (emptyselection) is)

`what happens if [] you go to home of this line`

Reply:
original:
this is an example
this line is selected [] lines after
blah blah blah


correct:
this is an example
[]this line is selected  lines after
blah blah blah


I think this is what you have:
this is an example
[]this line is selected 
 lines after
blah blah blah

Reply:
Please see the test example

Reply:
ah my example is not right then. It should look like this



original:
```
this is an example
this line is selected [pooh bear] lines after
blah blah blah
```

correct:
```
this is an example
[]this line is selected pooh bear lines after
blah blah blah
```

I think this is what you have:
```
this is an example
[]this line is selected
pooh bear lines after
blah blah blah

```

Reply:
fixed